src/Authenticator/HttpDigestAuthenticator.php

```diff
@@ -114,7 +114,7 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
      * Gets the digest headers from the request/environment.
      *
      * @param \Psr\Http\Message\ServerRequestInterface $request The request that contains login information.
-     * @return array Array of digest information.
+     * @return array|null Array of digest information.
```

Comment:
Do we need the null alternative here? I would simply do `if (!$result)` to check for emptiness - the strict ones can use `$result === []`. The less return types the better, not?

Reply:
You had the same question on the original ticket:
https://github.com/cakephp/authentication/pull/175#pullrequestreview-84302555
Is the conclusion still the same?

Reply:
OK, sry, didnt remember that one.